kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/api/common/LocalRef.java

```diff
@@ -40,4 +45,14 @@ public final boolean equals(Object obj) {
                 && Objects.equals(getKind(), other.getKind())
                 && Objects.equals(getName(), other.getName());
     }
+
+    @Override
+    public String toString() {
+        String group = getGroup().isEmpty() ? "" : "." + getGroup();
+        return getKind().toLowerCase(Locale.getDefault()) + group + "/" + getName();
```

Comment:
Use Locale.ROOT, I don't think it is right to consider kinds etc as natural language.